opennms-web-api/pom.xml

```diff
@@ -85,6 +85,14 @@
       <artifactId>spring-web-dependencies</artifactId>
       <type>pom</type>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.features.enlinkd</groupId>
+      <artifactId>org.opennms.features.enlinkd.persistence.api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.opennms.features.enlinkd</groupId>
+      <artifactId>org.opennms.features.enlinkd.service.api</artifactId>
+    </dependency>
```

Comment:
Can the corresponding deps be removed from the `opennms-webapp` pom now?

Reply:
Good catch, Thanks!  Please see the update.